pom.xml

```diff
@@ -66,7 +66,8 @@
         <jose4j.version>0.9.6</jose4j.version>
 
         <!-- Test dependencies -->
-        <strimzi.version>0.45.0</strimzi.version>  <!-- when bumping strimzi, be sure to comment out the kafka.version override in the systemtest module's pom -->
+        <strimzi.version>0.45.0
+        </strimzi.version>  <!-- when bumping strimzi, be sure to comment out the kafka.version override in the systemtest module's pom -->
```

Comment:
Was there a reason for the new line changes in this file?

Reply:
No, just an errant formatter. Will revert